build/jakarta/optaplanner-quarkus3.yaml

```diff
@@ -8,9 +8,14 @@ tags:
   - quarkus
   - jakarta
 recipeList:
+  # TODO: maybe replace by changing the revapi.oldOptaplannerVersion property to the 9.x version
+  - org.openrewrite.maven.ChangePluginConfiguration:
+      groupId: org.revapi
+      artifactId: revapi-maven-plugin
+      configuration: <skip>true</skip>
```

Comment:
As long as we explain that Revapi is still checked on 8.x and there are no differences between 8.x and 9.x, I think that this TODO may just go away.

Reply:
That's an option, yes.

Reply:
I will keep the TODO note for now and decide after we have the 9.x branch. In case the branches diverge over time, the revapi check could be useful.

Reply:
I see your point, I just want to say that the branches *must not diverge*. That's the whole goal of doing this big massive migration effort, instead of just branching.

Once the branches diverge, 8 needs to be abandoned.